spec/food_truck_spec.rb

```diff
@@ -0,0 +1,31 @@
+require './lib/item'
+require './lib/food_truck'
+
+RSpec.describe FoodTruck do
+  before(:each) do
+    @food_truck = FoodTruck.new("Rocky Mountain Pies")
+    @item1 = Item.new({name: 'Peach Pie (Slice)', price: "$3.75"})
+    @item2 = Item.new({name: 'Apple Pie (Slice)', price: '$2.50'})
+  end
+
+  describe '#initialize' do
+    it 'has attributes' do
+      expect(@food_truck.name).to eq("Rocky Mountain Pies")
+      expect(@food_truck.inventory).to eq({})
+      expect(@food_truck.check_stock(@item1)).to eq(0)
+    end
+  end
+  
+  describe '#stock' do
+  it 'stocks the inventory' do
+    @food_truck.stock(@item1, 30)
+    expect(@food_truck.inventory).to eq({@item1 => 30})
+    expect(@food_truck.check_stock(@item1)).to eq(30)
+    @food_truck.stock(@item1, 25)
+    expect(@food_truck.check_stock(@item1)).to eq(55)
+    @food_truck.stock(@item2, 12)
+    expect(@food_truck.check_stock(@item2)).to eq(12)
+    expect(@food_truck.inventory).to eq({@item1 => 55, @item2 => 12})
+    end
+  end
+end
```

Comment:
I encourage you to use white line spaces within your tests. LInes 20-28 are just a sea of text. I'd put a space after line 21, line 23, line 24, line 25, and line 26 to show the sections of the test. 

Great job testing that we can not only stock items, but we can stock the same item multiple times so we know it adds the quantities together instead of over-ridding the amount.